src/RoverExplorer/RoverExplorer.module.scss

```diff
@@ -1,3 +1,99 @@
-.roverExplorer {
-    background-color: blue;
-}
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Rajdhani:wght@700&display=swap');
+@import "../generalStyle.scss";
+
+@mixin GridItem{
+    align-items: center; 
+    text-align: center;
+     vertical-align: middle; 
+     line-height: 200px;
+      font-size: 20px; 
+      color: white;
+      border-radius: 3%; 
+      position: absolute;
+      top: 0; left: 0;
+      width: 100%; height: 100%;
+      border-color: white;
+      border-style: inset;
+      border-width:0.1px;
+      height: 100%;
+      background-repeat: no-repeat;
+      background-size: cover;
+      background-position: center;
+      background-color: grey;
+      @include subtitle;
+}
+
+
+.GridItem, img {
+    height: 100%;
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    background-position: center;
+}
+
+.Body, html {
```

Comment:
this will set the background for all the pages!
probably want to use class names instead.